cohere/client.py

```diff
@@ -503,21 +504,83 @@ def create_cluster_job(
         threshold: Optional[float] = None,
         min_cluster_size: Optional[int] = None,
     ) -> CreateClusterJobResponse:
+        """Create clustering job.
+
+        Args:
+            embeddings_url (str): File with embeddings to cluster.
+            threshold (Optional[float], optional): TODO. Defaults to None.
```

Comment:
why the TODO?

Reply:
not sure how to word it properly, can you please help with description?

Reply:
how's this?
```suggestion
            threshold (Optional[float], optional): Similarity threshold above which two texts are deemed to belong in the same cluster. Defaults to None.
```

Reply:
very nice!